rerun_py/rerun_sdk/rerun/log/image.py

```diff
@@ -90,6 +101,22 @@ def log_image(
     if interpretable_as_image and num_non_empty_dims != len(shape):
         image = np.squeeze(image)
 
+    if jpeg_quality is not None:
+        try:
+            pil_image = Image.fromarray(image)
+            output = BytesIO()
+            pil_image.save(output, format="JPEG", quality=jpeg_quality)
+            jpeg_bytes = output.getvalue()
+            output.close()
+
+            # TODO(draw_order)
+            log_image_file(
+                entity_path=entity_path, img_bytes=jpeg_bytes, img_format=ImageFormat.JPEG, timeless=timeless
+            )
+            return
+        except ImportError:
+            _send_warning("Ignoring jpeg_quality because opencv-python is not installed", 1, recording=recording)
```

Comment:
Should this still be behind this try clause? If it should, warning string should be updated

Reply:
nope, that's a leftover. good catch